### ISSUE TITLE
Incompatibility with recent collective.js.datatables

### ISSUE THREAD

**Issue by keul** [plomino/Plomino#698]
Latest collective.js.datatables (3+) are incompatible with Plomino.

Activating the advanced table on view lead to weird effects:

* no checkbox column
* a strange text column with uid has been added

**Comment by ebrehault** [comment]
Yes, but it is connected to the JQueryUI version and to the JQuery version.
If you succeed in finding the right versions for those 3 components, please tell me.

Note: in Plomino 2 (Plone 5), all the js is based on patternlibs components, it is much easier to manage.
